TourMate/TourMate/Frontend/Plan/Views/PlansViews/PlansListView.swift

```diff
@@ -24,7 +24,7 @@ struct PlansListView: View {
     }
 
     var body: some View {
-        LazyVStack {
+        ScrollView {
```

Comment:
Cannot use Lazy stack as now it will keep fetching from DB when the plan view is shown